### DISCUSSION TITLE
Linking notices

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#688] [Q&A] [answered]
In the workshop on September 26, it was commented the following about linking notices:

eForms are linked by procedure ID but for specific cases: 
- Planning/PINs are not part of procedures: use BT-125 
- Awards within a framework contract: use OPT-100 
- Change notice: use BT-758 
- Contract modification: use BT-1501 
- Notices before eForms (and any other cases): use OPP-090

Karl Ferrand spoke on the subject but it was not clear to me.

- Planning/PINs are not part of procedures -> what does it mean "are not part of procedures"? Should type 16 subtypes that have previously had a PIN be aligned? Or only for some type of PINs?
- Awards within a framework contract: use OPT-100 -> Do the framework agreements or awards have to be linked? How should it be done?
- Notices before eForms (and any other cases): use OPP-090 -> In our development we have done it using the BT-125(i)-Lot and the NoticeDocumentReference node appears correctly in the physical model. Does it need to be done through the OPP-090?

Thanks in advance

**Comment by YvesJo** [comment]
1. when notices belong to the same Procedure, they share the same Procedure ID (BT-04). BT-04 does not exist for Planning notices and procedures actually start with Competition notices. Planning notices and their Parts may be combined to define lots in different procedures.
2. For a contract within a FA it is expected to refer to the result notice of the same procedure where the Framework contract is actually signed
3. BT-125 is exclusively for referencing Planning (PIN Only) notices and shouldn't be used for other purpose. From a Rsult notice in eforms it may be necessary to associate it to a TEDXML competition notice which does not have a procedure ID and in this case OPP-090 has to be used.

For further details and explanation and for non-technical questions please get in touch directly with the helpdesk

**Reply by AlexAS81** [reply]
Could you show us an example of this type?

For a contract within a FA it is expected to refer to the result notice of the same procedure where the Framework contract is actually signed.

Thanks

**Reply by YvesJo** [reply, marked as answer]
Please get in touch with the helpdesk for these non-technical questions
